src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
- [ ] Kindly convert the Calculator component to a functional-based instead of class-based as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components).